crates/typst/src/foundations/array.rs

```diff
@@ -485,11 +486,17 @@ impl Array {
         /// The real arguments (the other arguments are just for the docs, this
         /// function is a bit involved, so we parse the arguments manually).
         args: &mut Args,
+        #[external]
```

Comment:
I don't think this needs `#[external]` and manual parsing. That's only necessary for the other argument because it has special handling.